src/interfaces.ts

```diff
@@ -141,6 +141,7 @@ export interface IServerOptions {
   protocol: "http" | "https";
   resources: typeof Resource[];
   services?: Service[];
+  exception?: ExceptionLayer;
```

Comment:
This should be the `IExceptionLayer`. With that, I can create my own exception layer that does not extend the Drash layer, but rather completely overrides it.

A simple test with this would be a nice touch 😃

Reply:
in last commit, I use typeof ExceptionLayer (like 'typeof Resource').

I added a test, I don't know if it's that you want.

Reply:
@Guergeiro , i tested using `IExceptionLayer` as the expected type in the TS playground. seems to require `new MyExceptionClass()` to be used. see [playground here](https://www.typescriptlang.org/play?strict=false#code/JYOwLgpgTgZghgYwgAgJIFEAeSAOZgD2IAMnAJ7TIDeAUMsgnGAgBYAUEAXMgM5hSgA5gEpuANwLAAJgG4aAXxo0EAGzg8eyALJksufEVIUoyYAFscKiGYjhNGbBDyES5SrXqNm7Lr35DRZAlpajp6BiIeAisAOhUCQQ5hMMVFGlBIWEQUVAB5Z0jQ+ghHApBuB30XI2gFJVV1TQBlaDF3MIIyngr8gxAeOU9I-gBXBDACKDZOvu60XpceYSLwsBZgHhiZxeQAXmRtyMHkVPrh5DA95BAIAHdkFqg2qY9kEqqibh09Jz6aqAUwiAA).

Reply:
Yes. It's supposed to. Here's a [link](https://www.typescriptlang.org/play?strict=false#code/JYOwLgpgTgZghgYwgAgJIFEAeSAOZgD2IAMnAJ7TIDeAUMsgnGAgBYAUEAXMgM5hSgA5gEpuANwLAAJgG4aAXxo0EAGzg8eyALJksufEVIUoyYAFscKiGYjhNGbBDyES5SrXqNm7Lr35DRZAlpajp6BiIeAisAOhUCQQ5hMMVFZTUNbTIAYQBXPgIzPScDV2NkCExIEClNHWLnQzcTDwYmVg5uPgEQEVDw3lycaBivDohhOQGESOiIOIS2ACI4GEgoJeT6VKVQdfgkNAB5RpBNVsr9F24QCAB3ZDYY57goQR5uOBAyAG0AXWEyAAvAA+NANUpGaAKJSqdSaADK0DE7jCBFOH2OGKmETO-FyCDABCgbHRpUxqBO5MBrXoYBYwB4MTJLk0QOQLMiOJ2ykiYGQ-PZtweSKgKJJF0cp249SlkOaCmSvLxAoATMDkMLkKLxWxJVciDKcvkiUU5S4oVBFUA) with the same code, but the correct options signature.

Reply:
ahh constructor. ok